package/src/headless/index.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import CanvasKitInit from "canvaskit-wasm/bin/full/canvaskit";
```

Comment:
Should this file be exported at a higher level to avoid deep imports?

Reply:
What do you mean?

Reply:
I'm not sure what I mean here, so I'll just mark it as resolved... Don't remember 🤪